Makefile

```diff
@@ -227,7 +227,7 @@ release: kustomize
 	mkdir build
 	cd details/operator-sdk/config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	cd details/operator-sdk && $(KUSTOMIZE) build config/default > $(BUILD_DIR)/only_astraconnector_operator.yaml
-	cat $(MAKEFILE_DIR)/details/operator-sdk/config/samples/neptune.yaml > $(BUILD_DIR)/astraconnector_operator.yaml
+	cat $(MAKEFILE_DIR)/details/operator-sdk/config/samples-neptune/neptune.yaml > $(BUILD_DIR)/astraconnector_operator.yaml
```

Comment:
This `neptune.yaml` was in a folder that already included a `kustomization.yaml` but wasn't actually included as a resource, giving me two options:
1. Add `neptune.yaml` as a resource to the original kustomization file and potentially disrupt whatever was making use of that kustomization file
2. Move `neptune.yaml` to a new folder so that I can create a new kustomization file

I chose #2 since it didn't seem like anything other than this Make target was making use of `neptune.yaml` right now. LMK if that assumption was wrong.

Reply:
I think moving it out of the /samples directory was the right move here as I am not sure why it was placed there to begin with. One suggestion would be to just call the new directory "neptune" as it doesn't contain any sample CRs. One other place that will need to be updated is this: https://github.com/NetApp-Polaris/neptune/blob/main/.github/workflows/post-merge.yaml#L103

Reply:
Neptune PR: https://github.com/NetApp-Polaris/neptune/pull/696